src/rtlib/dos/symb_reg.txt

```diff
@@ -0,0 +1,1661 @@
+extern_asm(_fb_WstrInstrAny);

```

Comment:
Can the tool to generate this file be included in the source tree, so others can use it to update this file (or is it not supposed to be changed)? Can this file be automatically generated by the makefile?

Reply:
The tool consists into a .bat file, and an executable (with .bas source included), built by Monochromator. I guess it could be included, but comments and texts are in a language I can't identify. I had to use that tool to rebuild symb_reg.txt, since it had to be changed from Freebasic 0.90. I can provide you the files... not sure how to integrate it in the makefile

Reply:
Actually, I checked, and it cannot be automatically generated by the makefile. In fact, the program extract the symbol names from the file libfb.dxe, that is built from libfb.a . Basically, it needs a copy of the runtime library that is already compiled, then it generates the updated symbol list (at this point, it's possible to replace the file symb_reg.txt with the updated one, delete the .o files and rebuild the library). So, it can't be automated in the makefile.

Reply:
Ah, ok thanks. Could you please add the maksymbr.bas and bat script to src/rtlib/dos, and translate it to English if at all possible? It seems very important to have in the repository together with the feature, since I suppose it will have to be updated everytime the public interface of libfb is changed.

Reply:
Translated.... the characters seemed to be Cyrillic, and it was in Russian. I used Google translator, for it